Content.Server/AI/AimShootLifeProcessor.cs

```diff
@@ -20,9 +21,9 @@ namespace Content.Server.AI
     [AiLogicProcessor("AimShootLife")]
     class AimShootLifeProcessor : AiLogicProcessor
     {
-        private readonly IPhysicsManager _physMan;
-        private readonly IServerEntityManager _entMan;
-        private readonly IGameTiming _timeMan;
+        [Dependency, UsedImplicitly] private readonly IPhysicsManager _physMan;
```

Comment:
I personally haven't had any issues using `#pragma warning disable/restore 0649` to handle DI fields. Both Rider and Roslyn don't give warnings about not assigned but they will keep warning about not being used at all.

Reply:
Yeah, Travis is still reporting a ton of warnings. You definitely should fix those.